rust/rpc/src/lib.rs

```diff
@@ -271,6 +273,10 @@ impl<W: Write + Send> RpcLoop<W> {
                     let json = match self.reader.next(&mut stream) {
                         Ok(json) => json,
                         Err(err) => {
+                            if self.peer.0.is_blocked.load(Ordering::SeqCst) {
```

Comment:
Okay, so this actually made me do some reading on the various orderings, and I think we have another option: this load should just `Acquire`, and the store below should use `Release`. Sorry for the confusion. :)

Reply:
Done